tests/asserts/keywords.py

```diff
@@ -1159,6 +1160,7 @@ def strat_err_fn(msg):
                                     msg = f"🔴ERROR: Strat name contains '{phrase}' but there is no {entrance_condition_name} entrance condition:{stratRef}"
                                     messages["reds"].append(msg)
                                     messages["counts"]["reds"] += 1
+                                break
```

Comment:
This deals with "Come In Shinecharged Jumping" and "Come In Shinecharged" containing the same string.  The phrase and requirement matching still happens once, so it seems fine.  Maybe some weirdness happens later with more related phrase entries.